pliers/support/exceptions.py

```diff
@@ -36,6 +36,9 @@ class MissingDependencyError(PliersError):
     """
 
     def __init__(self, dependencies, message=MISSING_DEPENDENCY_MESSAGE,
-                 *args, **kwargs):
-        msg = message % ', '.join(dependencies)
+                 custom_message=None, *args, **kwargs):
+        if custom_message:
```

Comment:
You probably already know this, and I don't actually have any objection to the more explicit syntax, but you could use a ternary conditional here: 
```python
msg = custom_message if custom_message else message % ', '.join(dependencies)
```

Actually, in this particular case, you can even write:
```python
msg = custom_message or message % ', '.join(dependencies)
```